daemon/network.go

```diff
@@ -338,11 +339,12 @@ func (daemon *Daemon) GetNetworkDriverList() map[string]bool {
 	networks := c.Networks()
 
 	for _, network := range networks {
-		driver := network.Type()
-		pluginList[driver] = true
+		pluginList = append(pluginList, network.Type())
```

Comment:
it gets a list of networks, not a list of plugins, so two networks with the same driver/plugin will result in duplicate entries. I think that's the reason the map was used originally. Also note this PR, which does something similar: https://github.com/docker/docker/pull/25036

Perhaps we should have something for this ("unique sorted list") that we can reuse